src/ql/arch/cc/pass/gen/vq1asm/detail/operands.h

```diff
@@ -0,0 +1,149 @@
+/** \file
```

Comment:
Why is this not common code?

Reply:
two lines below there's this note:
`FIXME: could be useful for other backends and should be moved if this becomes appropriate
`
But making is common may require more generality than currently present.

There is also overlap with new_to_old.cc, but since that should eventually be deprecated it doesn't seem worth the effort.